index.html

```diff
@@ -1,57 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Calculator | Dhanush Nehru</title>
 </head>
+
 <body>
-<div class="container">
-    <h1 align="center">
-        Calculator
-    </h1>
-</div>
-<div class="calculator">
-    <input type="text" id="screen" maxlength="30"/>
-    <div class="calc-buttons">
-        <div class="functions-one">
-            <button class="button numbers"><h1>7</h1></button>
-            <button class="button numbers"><h1>8</h1></button>
-            <button class="button numbers"><h1>9</h1></button>
-            <button class="button numbers"><h1>4</h1></button>
-            <button class="button numbers"><h1>5</h1></button>
-            <button class="button numbers"><h1>6</h1></button>
-            <button class="button numbers"><h1>1</h1></button>
-            <button class="button numbers"><h1>2</h1></button>
-            <button class="button numbers"><h1>3</h1></button>
-            <button class="button triggers"><h1>C</h1></button>
-            <button class="button numbers"><h1>0</h1></button>
-            <button class="button basic-stuff"><h1>.</h1></button>
-        </div>
-        <div class="functions-two">
-            <button class="button complex-stuff"><h1>%</h1></button>
-            <button class="button complex-stuff"><h1>x !</h1></button>
-            <button class="button complex-stuff"><h1>x^</h1></button>
-            <button class="button basic-stuff"><h1>x</h1></button>
-            <button class="button basic-stuff"><h1><span>&#247;</span></h1></button>
-            <button class="button basic-stuff"><h1>+</h1></button>
-            <button class="button complex-stuff"><h1>x <sup>2</sup></h1></button>
-            <button class="button basic-stuff"><h1>-</h1></button>
-            <button class="button complex-stuff"><h1><span>&#x221A;</span></h1></button>
-            <button class="button basic-stuff"><h1>(</h1></button>
-            <button class="button triggers"><h1>=</h1></button>
-            <button class="button basic-stuff"><h1>)</h1></button>
+    <div class="container">
+        <h1 align="center">
+            Calculator
+        </h1>
+    </div>
+    <div class="calculator">
+        <input type="text" id="screen" maxlength="30" />
+        <button class="back">Back</button>

```

Comment:
why you dont use the `button trigger` class? At least you should use the `button` class.